sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
Why are we adding this notebook? We removed all standalone -mlflow notebooks. automl-image-object-detection-task-fridge-items.ipynb has all the required changes.

Reply:
Removed. Was added erroneously, during merge.